beacon-chain/node/node_test.go

```diff
@@ -41,27 +41,6 @@ func TestNodeValidator_Builds(t *testing.T) {
 	}
 }
 
-// Test that beacon chain node can start.
-func TestNodeStart(t *testing.T) {
-	app := cli.NewApp()
-	set := flag.NewFlagSet("test", 0)
-	set.String("web3provider", "ws//127.0.0.1:8546", "web3 provider ws or IPC endpoint")
-	tmp := fmt.Sprintf("%s/datadir", os.TempDir())
-	set.String("datadir", tmp, "node data directory")
-	set.Bool("enable-powchain", false, "no powchain service")
-	set.Bool("demo-config", true, "demo configuration")
-
-	context := cli.NewContext(app, set, nil)
-
-	node, err := NewBeaconNode(context)
-	if err != nil {
-		t.Fatalf("Failed to create BeaconNode: %v", err)
-	}
-
-	go node.Start()
```

Comment:
The problem here is that this goroutine is still running during execution of the _next_ test. This isn't actually testing anything, so I just deleted it.